qa/standalone/scrub/osd-scrub-snaps.sh

```diff
@@ -121,12 +123,14 @@ function create_scenario() {
 
     # Starts with a snapmap
     ceph-kvstore-tool bluestore-kv $dir/${osd} list 2> /dev/null > $dir/drk.log
-    grep "^m.*SNA_.*[.]7[.]obj16[.][.]$" $dir/drk.log || return 1
+    grep SNA_ $dir/drk.log
```

Comment:
do we need this log?

Reply:
On failure it allows analysis of what happened.